demo/src/com/google/maps/android/utils/demo/GeoJsonDemoActivity.java

```diff
@@ -122,6 +122,13 @@ protected void onPostExecute(JSONObject jsonObject) {
                 // Add the layer onto the map
                 addColorsToMarkers();
                 mLayer.addLayerToMap();
+
+                mLayer.setOnFeatureClickListener(new GeoJsonLayer.GeoJsonOnFeatureClickListener() {
+                    @Override
+                    public void onFeatureClick(GeoJsonFeature feature) {
+                        Log.i(mLogTag, "Feature clicked: " + feature.getProperty("title"));
```

Comment:
Can you use a toast here instead? It'd be great to allow the user to see what's happening without having to fire up a log catcher.